Packs/CortexXDR/ReleaseNotes/6_0_3.md

```diff
@@ -0,0 +1,20 @@
+#### Playbooks
+
+##### Cortex XDR - Retrieve File Playbook
+
+- Deprecated. Use Cortex XDR - Retrieve File Playbook v2 instead.
+
+##### New: Cortex XDR - Retrieve File v2
+
+- New: Retrieves files from selected endpoints. You can retrieve up to 20 files, from no more than 10 endpoints.
+Inputs for this playbook are:
+- A comma-separated list of endpoint IDs.
```

Comment:
Combine these 2 lines into one, so the MD will be much more readable for the user.